test/specs/node/cutlass/basic_spec.rb

```diff
@@ -1,75 +0,0 @@
-# frozen_string_literal: true
-
-require_relative "../spec_helper"
-
-
-describe "Heroku's Nodejs CNB" do
-  ["heroku/builder:20", "heroku/builder:22"].each do |builder|
-    describe "with #{builder}" do
-      it "handles a downgrade of node engine" do
-        Cutlass::App.new("simple-function", builder: builder).transaction do |app|
-          set_node_version_in_dir(app.tmpdir, version: "^14.0")
-
-          app.pack_build do |pack_result|
-            expect(pack_result.stdout).to include("Installing Node.js 14.")
```

Comment:
We didn't port this test `set_node_version_in_dir` logic to the new tests. Is that intentional?

Reply:
i did port that over, yes.  i mistakenly added it to the previous PR (#663) though.  the test is [here](https://github.com/heroku/buildpacks-nodejs/blob/main/buildpacks/nodejs-engine/tests/integration_test.rs#L30-L49)